templates/eventstore/0/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '2'
+services:
+  eventstore:
+    image: eventstore/eventstore:4.0.3
```

Comment:
Looking at https://hub.docker.com/r/eventstore/eventstore/tags/ seems that image tag doesn't exist...Please correct it. Tested and working with this image tag...
```
-    image: eventstore/eventstore:4.0.3
+    image: eventstore/eventstore:release-4.0.3
```